src/api/goods.ts

```diff
@@ -10,10 +10,13 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => goods
+      .sort((g1, g2) => g1.name.localeCompare(g2.name))
```

Comment:
it's not a big mistake, but it's better to rename let g1 and g2 to good1 and good2

Reply:
Agree with @erikkaraman )